docs/content/en/docs/getting-started/baremetal/bare-spec.md

```diff
@@ -185,7 +185,7 @@ You can omit `workerNodeGroupConfigurations` when creating Bare Metal clusters.
 >**_NOTE:_** Empty `workerNodeGroupConfigurations` is not supported when Kubernetes version <= 1.21.
 
 ### workerNodeGroupConfigurations[*].count (optional)
-Number of worker nodes. Optional if autoscalingConfiguration is used, in which case count will default to `autoscalingConfiguration.minCount`.
+Number of worker nodes. It will be ignored if the [cluster autoscaler curated package]({{< relref "../../packages/cluster-autoscaler/addclauto" >}}) is installed and autoscalingConfiguration is used to specify the desired range of replicas.
```

Comment:
We should still call this out as optional only if autoscalingConfiguration is used, because we will fail validation if not as it is required otherwise.
```suggestion
Number of worker nodes. It will be ignored if the [cluster autoscaler curated package]({{< relref "../../packages/cluster-autoscaler/addclauto" >}}) is installed and `autoscalingConfiguration` is used to specify the desired range of replicas.
```

Reply:
It's an optional field even if autoscalingConfiguration is not used as it will default to `1` if not specified. We won't fail any validation if it is not specified.